src/YoloDev.Expecto.TestSdk/settings.fs

```diff
@@ -14,58 +18,159 @@ type RunSettings =
     disableParallelization: bool
 
     /// Gets a value which indicates the target framework the tests are being run in.
-    targetFrameworkVersion: string option }
+    targetFrameworkVersion: string option 
+
+    /// Gets the [ExpectoConfig](https://github.com/haf/expecto#the-config) that was set via RunSettings
+    expectoConfig : ExpectoConfig}
```

Comment:
Nitpicking, but please try to keep to the style used in this project wrt spaces around symbols.

I won't guarantee that I've been consistent, but in general at least, no spaces in front of colon (in record types), and always a space before `}`.

Reply:
👍